apps/meteor/app/federation-v2/server/infrastructure/matrix/Bridge.ts

```diff
@@ -146,6 +146,33 @@ export class MatrixBridge implements IFederationBridge {
 		}
 	}
 
+	public async verifyInviteeId(externalInviteeId: string): Promise<string> {
+		const [userId, homeserverUrl] = externalInviteeId.replace('@', '').split(':');
+		try {
+			const response = await fetch(`https://${homeserverUrl}/_matrix/client/v3/register/available?username=${userId}`);
+
+			if (response.status === 400) {
+				const responseBody = await response.json();
+
+				if (responseBody.errcode === 'M_USER_IN_USE') {
```

Comment:
Wouldn't be better to have these comparisons as constants, (e.g `const userExists = responseBody.errcode === 'M_USER_IN_USE'`)? Instead of having code comments explaining them?

Reply:
added the constant here: [apps/meteor/app/federation-v2/server/infrastructure/helpers/MatrixIdVerificationTypes.ts](https://github.com/RocketChat/Rocket.Chat/pull/28096/files#diff-58880fcef31d139da50a34bbfc71cd6db076bfcdb96dd47f55523d60249abb5d)